docs/resources/vpcep_service.md

```diff
@@ -56,8 +56,12 @@ The following arguments are supported:
 
 * `approval` - (Optional, Bool) Specifies whether connection approval is required. The default value is false.
 
-* `permissions` - (Optional, List) Specifies the list of accounts to access the VPC endpoint service. The record is in
-  the `iam:domain::domain_id` format, while `*` allows all users to access the VPC endpoint service.
+* `permissions` - (Optional, List) Specifies the list of accounts to access the VPC endpoint service.
+  The record is in the `iam:domain::domain_id` format, while `*` allows all users to access the VPC endpoint service.
+
+* `organization_permissions` - (Optional, List) Specifies the list of organization to access the VPC endpoint service.
+  The record is in the `organizations:orgPath::org_path` format, while `*` allows all users who belong to the
+  organization to access the VPC endpoint service.
```

Comment:
The description is ambiguous. Word `the organization` is not exist when this field is set to `*`.